docs/en/operation/Tiered-Locality.md

```diff
@@ -93,18 +91,43 @@ If other locality tiers are left unset, they will not be used to inform locality
 
 ### Setting locality
 
-#### Locality script
+#### Option A: shell script
 
 By default, Alluxio clients and servers search the classpath for a script named
-`alluxio-locality.sh`.
-Output format of this script is a comma-separated list of tierName=tierValue pairs.
-The script name can be overridden by setting:
+`alluxio-locality.sh` (can be overridden by setting `alluxio.locality.script`).
+Output format of this script is a comma-separated list of `tierName=tierValue` pairs.
+For example, suppose Alluxio workers are spread across multiple availability zones within EC2.
+To configure tiered locality with availability zones:
 
-```
-alluxio.locality.script=locality_script_name
+1. On the Alluxio master(s), set `alluxio.locality.order=node,availability_zone` to define the order
+   of locality tiers to prefer `node` over `availability_zone`.
+
+1. Create script `alluxio-locality.sh` to determine the tiered identity for each entity.
+   The output format is a comma-separated list of `tierName=tierValue` pairs.
+   ```bash
+   #!/usr/bin/env bash
+
+   node=$(hostname)
+   # Ask EC2 which availability zone we're in
+   availability_zone=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
+
+   echo "node=${node},availability_zone=${availability_zone}"
+   ```
+
+1. Make the script executable with `chmod +x alluxio-locality.sh`.
+
+1. Include the script on the classpath of your applications and Alluxio servers.
+For servers, put the file in the `${ALLUXIO_HOME}/conf/` directory.
+
+1. Restart Alluxio servers to pick up the configuration changes.
+
+A log entry should appear of the format:
+
+```log
+INFO  TieredIdentityFactory - Initialized tiered identity TieredIdentity(node=ip-xx-xx-xx-xx, availability_zone=us-east-1)
 ```
 
-#### Configuration properties
+#### Option B: configuration properties
 
 Using locality script is the preferred way to configure tiered locality
```

Comment:
Using the locality shell script is the preferred way to configure tiered locality over configuration properties because...

Reply:
done